### DISCUSSION TITLE
Transforming file name labels in diff mode

### DISCUSSION THREAD

**Discussion by salmankhilji** [dandavison/delta#997] [Q&A]
How do I hide _one_ of the filenames within the header when invoking `delta` with two files on the CLI?

i.e., by default, delta displays `A ⟶   B`.  I want to hide the ` ⟶   B` part.

```
$ delta -s --hunk-header-style=omit A B

A ⟶   B
────────────────────────────────────────────────────────────────────

│  1 │A                           │    │
│    │                            │  1 │B
```
Not sure how to use the `--file-transformation` option.  Here is how I tried to use it (unsuccessfully):

```
$ delta -s --hunk-header-style=omit --file-transformation 's/A/Z/' A B

A ⟶   B
────────────────────────────────────────────────────────────────────

│  1 │A                           │    │
│    │                            │  1 │B
```

**Comment by Gerst20051** [comment]
I am also trying to figure this out.

`/dev/fd/12 ⟶   /dev/fd/14`

 This would be useful when diffing like this:

`delta <(sort file1) <(sort file2)`

I got it to work like this:

`delta --file-style omit <(sort file1) <(sort file2)`

I didn't use `--hunk-header-style omit` because that removes the line numbers and I want to keep the line numbers.
